apiserver/grpc/server.go

```diff
@@ -58,11 +58,15 @@ func (g *GRPCServer) ApplyOneway(ctx context.Context, req *proto.ApplyPistageReq
 	if err != nil {
 		return nil, err
 	}
+
+	ctx, cancel := context.WithTimeout(ctx, g.timeout)
+	defer cancel()
+
```

Comment:
Yes, cannot defer and cannot use the `ctx` passed into this function. I had tested the context. If using the `ctx`, the task will be canceled immediately.